src/platform/forms-system/src/js/components/FormSignature.jsx

```diff
@@ -114,12 +114,15 @@ export const FormSignature = ({
         required={required}
         errorMessage={(showError || signature.dirty) && signatureError}
       />
-      <Checkbox
+      <va-checkbox
         label={checkboxLabel}
+        description={null}
```

Comment:
If there isn't anything being passed into the `description` prop, can we just remove it?

Reply:
In [the docs](https://design.va.gov/storybook/?path=/docs/components-va-checkbox--default#with-description-jsx) and when I was on the FLT team, I remember we needed to keep the `description` prop specifically set to null. I think the reason was that if we know there won't be a prop for `description` (because we can't guarantee description is a string, like when we need to use an anchor tag), then we will use a slot to show `description`. But if we don't explicitly set the `description` prop to null, and we inherit a bunch of properties and apply them to a component (see `{...field}` in image below -- taken from FLT repo)
![image](https://user-images.githubusercontent.com/15200011/221928351-ba0b89fc-bda0-49be-8116-265b8fd1e2c2.png)
, there's a chance we can inherit a description property. If we do that, then we override any slotted description, so I think it's basically a failsafe to make sure we don't accidentally override our slotted description.

Reply:
@nickjg231 Is there an opportunity here to use the `va-privacy-policy` component instead of `va-checkbox` for some of these swaps? We have a specific component for that if that's what some of these checkboxes were trying to emulate:

https://design.va.gov/storybook/?path=/docs/components-va-privacy-agreement--default

![Screenshot 2023-02-28 at 4 26 20 PM](https://user-images.githubusercontent.com/872479/221995717-73cd7cf0-a27e-407a-af54-c11614bb201b.png)